src/app/containers/ComscoreAnalytics/Canonical/index.jsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable jsx-a11y/alt-text */
+import React, { useContext, useEffect } from 'react';
+import { Helmet } from 'react-helmet';
+import { UserContext } from '#contexts/UserContext';
+
+const CanonicalComscoreAnalytics = () => {
+  const { personalisationEnabled } = useContext(UserContext);
+
+  const staticAssetsPath = `${process.env.SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN}${process.env.SIMORGH_PUBLIC_STATIC_ASSETS_PATH}`;
+
+  useEffect(() => {
+    const csUcfr = personalisationEnabled ? '1' : '0';
+    /* eslint-disable no-underscore-dangle */
+    window._comscore = window._comscore || [];
+    window._comscore.push({ c1: '2', c2: '17986528', cs_ucfr: csUcfr });
+  }, [personalisationEnabled]);
+
+  return (
+    <Helmet>
+      <script
+        async
+        type="text/javascript"
+        src={`${staticAssetsPath}static/js/comscore-1.0.js`}
+      />
+      <noscript>
+        {`<img src="https://sb.scorecardresearch.com/p?c1=2&c2=17986528&cs_ucfr=0&cv=2.0&cj=1" />`}
```

Comment:
```suggestion
        {`<img src="https://sb.scorecardresearch.com/p?c1=2&c2=17986528&cs_ucfr=${csUcfr}&cv=2.0&cj=1" />`}
```

Reply:
I've set this to 0 (non-personalised) here because we can't determine whether the user has `personalisationEnabled` on our server. Therefore we do not want to send a value of 1 if we can not 100% confirm fi they personalised.

Reply:
Ah ok, just read this comment https://github.com/bbc/simorgh/pull/7391#discussion_r465804672 👍